app/src/main/java/edu/agh/susgame/front/Config.kt

```diff
@@ -12,7 +12,18 @@ object Config : AppConfig {
         // NOTE: This value must be changed in `app/src/main/res/xml/network_security_config.xml` as well
         domain = "192.168.0.105",
         port = 8080,
-
         timeout = 5.seconds,
     )
+
+    override val gameConfig = AppConfig.GameConfig(
+        maxPinLength = 6,
+        gameTimeMinutes = AppConfig.GameConfig.GameTimeMinutes(
+            min = 5,
+            max = 15
+        ),
+        playersPerGame = AppConfig.GameConfig.PlayersPerGame(
+            min = 2,
+            max = 6
+        ),
+    )
```

Comment:
Bardzo fajnie, że to wyjąłeś. Tutaj, żeby trochę ograniczyć boilerplate, można sobie pomóc importami:

```
Index: app/src/main/java/edu/agh/susgame/front/Config.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/agh/susgame/front/Config.kt b/app/src/main/java/edu/agh/susgame/front/Config.kt
--- a/app/src/main/java/edu/agh/susgame/front/Config.kt	(revision 7b0bf081545b1397e366c8d404820788c0c3b64a)
+++ b/app/src/main/java/edu/agh/susgame/front/Config.kt	(date 1718525196243)
@@ -1,13 +1,15 @@
 package edu.agh.susgame.front
 
 import edu.agh.susgame.front.util.AppConfig
+import edu.agh.susgame.front.util.AppConfig.*
+import edu.agh.susgame.front.util.AppConfig.GameConfig.*
 import edu.agh.susgame.front.util.ProviderType
 import kotlin.time.Duration.Companion.seconds
 
 object Config : AppConfig {
     override val providers = ProviderType.MockLocal
 
-    override val webConfig = AppConfig.WebConfig(
+    override val webConfig = WebConfig(
         protocol = "http",
         // NOTE: This value must be changed in `app/src/main/res/xml/network_security_config.xml` as well
         domain = "192.168.0.105",
@@ -15,13 +17,13 @@
         timeout = 5.seconds,
     )
 
-    override val gameConfig = AppConfig.GameConfig(
+    override val gameConfig = GameConfig(
         maxPinLength = 6,
-        gameTimeMinutes = AppConfig.GameConfig.GameTimeMinutes(
+        gameTimeMinutes = GameTimeMinutes(
             min = 5,
             max = 15
         ),
-        playersPerGame = AppConfig.GameConfig.PlayersPerGame(
+        playersPerGame = PlayersPerGame(
             min = 2,
             max = 6
         ),
```